utils/flare.py

```diff
@@ -94,11 +94,11 @@ class Flare(object):
     ]
     MAIN_CREDENTIALS = [
         CredentialPattern(
-            re.compile('^\s*api_key:( *\w+(\w{5}) ?,?)+$'),
+            re.compile('^\s*api_key\s*[=, :]( *\w+(\w{5}) ?,?)+$'),
```

Comment:
I think this piece of code is overly complex, your patch is good but I'd make it a little bit simpler.

1) we can change the regex to only capture in a group the last 5 digits of the api key 
2) instead of nesting maps and lambdas, proceed with a string replace using a backreference to the group from above

the following piece of code should do the same thing (untested):
```
CredentialPattern(
  re.compile(r'^\s*api_key\s*[=, :]\s*\w{27}(\w{5})$'),
  r'api_key: ***************************\1',
  'api_key'
)
```

Reply:
I definitely agree here. Removing the lambdas and just replacing everything but the last 5 chars does indeed do the trick. Thanks!

Reply:
Looks like it doesn't work for comma separated API keys, left more details in the notes based on our discussions